usecases/service/group.go

```diff
@@ -50,14 +50,14 @@ func (s *groupService) GetGroup(ctx context.Context, groupID uuid.UUID) (*domain
 	// fill members info
 	for i, v := range groups.Members {
 		if u, ok := umap[v.User.ID]; ok {
-			groups.Members[i].User = *domain.NewUser(u.ID, u.Name, u.RealName, u.Check)
+			groups.Members[i].User = *domain.NewUser(u.ID, u.Name, u.RealName(), u.Check)
```

Comment:
Check=falseのときにrealNameが空になっちゃうので`u`をコピーして使うのがいいと思います
これをするとテストも`random.Bool()`でよくなるはず
```suggestion
			m := *u
			groups.Members[i].User = m
```